packages/lockdown-shim/test/tame-error-allow.test.js

```diff
@@ -5,17 +5,39 @@ import { lockdown } from '../src/main.js';
 lockdown({ noTameError: true });
 
 test('lockdown allow Error - Error is not tamed', t => {
+  t.plan(6);
+
   const c = new Compartment();
   t.equal(c.evaluate('typeof Error.captureStackTrace'), 'function');
   t.equal(c.evaluate('typeof Error.stackTraceLimit'), 'number');
   t.equal(c.evaluate('typeof new Error().stack'), 'string');
-  t.end();
+
+  const error = new Error();
+  error.stack = false;
+  Error.captureStackTrace(error);
+  t.equal(typeof error.stack, 'string');
+  t.notEqual(error.stack, '');
+
+  t.throws(() => {
+    Error.stackTraceLimit = 10;
```

Comment:
Huh, if it's *not* tamed, then trying to set `.stackTraceLimit` throws, but if it *is* tamed, then trying to set it is silently ignored? I imagine there's some weird stuff going on here that I don't know about, but I'd kind of expect a non-tamed Error to allow setting of `.stackTraceLimit`.

Reply:
Do we have a document yet that explains each `lockdown` option, what you can/cannot do with the tamed/non-tamed versions?

Reply:
This PR allows you to run with taming of errors enabled. Did you create one for the old SES?

Reply:
That last bullet is now the experiment at https://github.com/jfparadis/SES-beta/pull/1

Reply:
Agreed
> This means that the untamed version doesn't really support the legacy operations

Reply:
I think we need a new option for this:
>In untamed (unsafe debug) mode only, exempt Error from the freezing of the primordial.

Taming and freezing are separate things. It would be much clearer in the code if we have an option and a path for `noHardenError` or something like that.

@erights thoughts?

Reply:
The winterize approach at https://github.com/jfparadis/SES-beta/pull/1 solves the problem, while still hardening the `Error` object. We don't need to introduce a `noHardenError` option.